code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -1,39 +1,22 @@
-/obj/structure/closet/secure_closet/captains
+/obj/structure/closet/secure_closet/captains //Unbloated by ShadeAware aka Michael Labs
```

Comment:
recommend changing comment to tell people that the other stuff is in vending/wardrobes